paperqa/llms.py

```diff
@@ -70,6 +75,24 @@ class EmbeddingModes(StrEnum):
     QUERY = "query"
 
 
+class OpenAIBatchStatus(StrEnum):
```

Comment:
Minor point, but we could make this a bit DRY-er by making a single BatchStatus(StrEnum), with methods that convert OpenAI and Anthropic statuses, so like: 
```python
class BatchStatus(StrEnum):
    COMPLETE = 'complete'
    PROGRESS = 'progress'
    ....
    @classmethod
    def from_anthropic(cls, status: str):
        return {'completed': cls.COMPLETE,
                    'in_progress': cls.PROGRESS, ...
                     }
```

Reply:
I think we want to do the opposite, right? We want to have something like `BatchStatus.from_<provider>.COMPLETE` that returns "completed" to openai and "ended" for anthropic. Maybe:

```python
class BatchStatus(StrEnum):
    COMPLETE = ""
    ...
    def from_openai(self) -> str:
        mapping = {
            BatchStatus.COMPLETE: "completed"
            ...
        }
        return mapping[self]
        
    def from_anthropic(self) -> str:
        mapping = {
            BatchStatus.COMPLETE: "ended"
            ...
        }
        return mapping[self]
```

then we can use it as `BathStatus.COMPLETE.from_openai()`